demo/validators.html

```diff
@@ -113,7 +113,9 @@
 
                 this.ajax = document.getElementById('ajax1');
                 this.ajax.addEventListener('response', function() {
-                  this.invalid = this.ajax.lastResponse.elements.indexOf(this.value) < 0;
+                  var elementName = this.value || '';
+                  this.invalid = this.ajax.lastResponse.elements.indexOf(elementName.toLowerCase()) < 0;
+                  this.errorMessage = this.invalid ? '"' + elementName + '" is not a Chemical Element' : '';
```

Comment:
This is exactly not how I see this attribute/property being used. The message should be set in advance declaratively, not programmatically after validation.